src/app/server/Mdns.cpp

```diff
@@ -247,22 +242,27 @@ CHIP_ERROR Advertise(bool commissionableNode)
 /// (Re-)starts the minmdns server
 void StartServer()
 {
-    ChipLogProgress(Discovery, "Start dns-sd server");
+    StartServer(false, false);
+}
+
+void StartServer(bool commissioningMode, bool additionalCommissioning)
+{
     CHIP_ERROR err = chip::Mdns::ServiceAdvertiser::Instance().Start(&chip::DeviceLayer::InetLayer, chip::Mdns::kMdnsPort);
 
-    // TODO: advertise this only when really operational once we support both
-    // operational and commisioning advertising is supported.
+    // TODO: advertise all operational ids once we support multiple
     if (GetCurrentNodeId() != kUndefinedNodeId)
     {
+        ChipLogProgress(Discovery, "Start dns-sd server CM=%d, AC=%d", commissioningMode ? 1 : 0, additionalCommissioning ? 1 : 0);
```

Comment:
These are the _A1 and _C1 bits from the spec, right?  Can we just label them that way in the logs?

Reply:
these are the _A1 is the query value while AC=1 is the key/value pair, so both are in the spec.

Reply:
There is no AC=1 in the spec.  There's AP=1....